libraries/chain/db_maint.cpp

```diff
@@ -465,9 +465,12 @@ void database::update_active_sons()
       else
       {
          vote_counter vc;
+         a.owner.account_auths.clear();
```

Comment:
Also update owner authority in if branch

Reply:
done.